core/server/utils/image-size-from-url.js

```diff
@@ -34,7 +31,7 @@ var sizeOf       = require('image-size'),
 module.exports.getImageSizeFromUrl = function getImageSizeFromUrl(imagePath) {
     return new Promise(function imageSizeRequest(resolve, reject) {
```

Comment:
As `got` uses promises, I don't think this needs to be wrapped anymore?

Reply:
Yeah this should work 👍

Reply:
Alright! Finally got it working! Change is pushed ;)